vsm

```diff
@@ -352,4 +452,10 @@ if __name__ == "__main__":
                 exit(1)
 
         state = State(args.initial_state, args.rules, log_categories)
+
+        log_replayer = None
+        if args.replay_log_file:
+            log_replayer = LogReplayer(state, args.replay_log_file,
```

Comment:
Why is it assigning to the `log_replayer` variable?

Apart of this detail, everything else looks good to merge.